tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+previous experience with Dart or mobile programming.
+
+* TOC
+{:toc}
+
+### What are Flutter and Dart?
+
+_Flutter_ is a mobile app SDK to help developers and designers
+build modern mobile apps for iOS and Android.
+
+_Dart_ is an application programming language from Google that's
+easy to learn, easy to use, and deployable everywhere.
+Flutter apps are written in Dart.
+
+### What you'll build
+
+You’ll implement a simple mobile app that proposes names for a
+startup company. The user can select and unselect proposed names,
+saving the best ones. The code generates ten names at a time.
+As the user scrolls, new batches of names are generated.
+
+<center><img src="images/StartupNameGenerator-screenshot.png" alt="Screenshot for a Startup Name Generator"></center>
+<center>[PENDING: Temp image - replace with video]</center><br>
+
+The following video shows how the finished app works:
+
+[PENDING: Matt is making the video]
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* Basic structure of a Flutter app.
+* Finding and using packages to extend functionality.
+* Using hot reload for a quicker development cycle.
+* How to implement a stateful widget.
+* How to create an infinite, lazily loaded list.
+* How to create and navigate to a second screen.
+* .
+
+</div>
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll use:</b>
+
+This tutorial requires version xxx or later of the Flutter SDK.
+When you install Flutter, you get the correct version of the
+Dart SDK as part of the install. You also get a Flutter Simulator,
+which you can use instead of a physical mobile device.
+
+These instructions feature the Android Studio IDE, but you can use whichever
+tools you prefer—Flutter and Dart plugins are also available for
+VSCode and JetBrains products.
+
+See [Mit's new setup page]() for information on how to set up
+your environment and create your first simple Flutter app.
+
+</div>
+
+## Step 1: Create the starting Flutter app
+
+Create a basic Flutter app, using the instructions in
+[Mit’s new page, preferably linked to a specific anchor for
+creating the templated app]().
+Name the project startup_namer. You’ll be modifying this starter
+app to create the finished app.
+
+### Update main.dart
+
+You'll update two files during this codelab. In the next step
+you'll update the pubspec file, `pubspec.yaml`, which manages
+the app's assets.
+You'll mostly be editing **lib/main.dart**, where the Dart code lives.
+
+<aside class="alert alert-success" markdown="1">
+<i class="fa fa-lightbulb-o"> </i> **Tip:**
+When pasting code into your app, indentation can
+become skewed. You can fix this in Android Studio by right-clicking the
```

Comment:
I would make this section a bit more general, like this:

```
You can fix this automatically with our tools:
  * Android Studio / IntelliJ IDEA: right-click the dart code and select  **Reformat Code with dartfmt**
  * VS Code: right-click the code and select **Format Document**
  * Terminal: run `flutter format <filename>`
```

Reply:
Done